plugin/signature_help.py

```diff
@@ -21,6 +21,22 @@
 from .core.settings import settings
 
 
+def get_documentation(d: 'Dict[str, Any]') -> 'Optional[str]':
```

Comment:
Unsure if this is the same code as the previous PR, but would be nice to have only 1 instance of this function in this codebase.

Reply:
This is commit c95ae70, cherry-picked from #520. I will address the rest of the review tomorrow.

Reply:
This has now been merged and thus is not visible anymore in this PR.